codegen/smithy-aws-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/aws/protocols/core/AwsHttpBindingProtocolGenerator.kt

```diff
@@ -39,7 +39,15 @@ abstract class AwsHttpBindingProtocolGenerator : HttpBindingProtocolGenerator()
         // The following can be used to generate only a specific test by name.
         // val targetedTest = TestMemberDelta(setOf("RestJsonComplexErrorWithNoMessage"), TestContainmentMode.RUN_TESTS)
 
-        val ignoredTests = TestMemberDelta(setOf())
+        val ignoredTests = TestMemberDelta(
+            setOf(
+                "AwsJson10ClientErrorCorrectsWithDefaultValuesWhenServerFailsToSerializeRequiredValues",
+                "RestJsonNullAndEmptyHeaders",
+                "NullAndEmptyHeaders",
+                "RpcV2CborClientPopulatesDefaultsValuesWhenMissingInResponse",
+                "RpcV2CborClientPopulatesDefaultValuesInInput",
```

Comment:
I remember seeing these changes and think we might actually have a bug here, fyi